src/js/profile/mobile/widget/DropdownMenu.js

```diff
@@ -970,11 +982,13 @@
 				self._nativeChangeOptionBound = nativeChangeOption.bind(null, self);
 				self._focusBound = onFocus.bind(null, self);
 				self._blurBound = onBlur.bind(null, self);
+				self._touchMoveBound = onTouchMove.bind(null, self);
 
 				elSelectWrapper.addEventListener("focus", self._focusBound);
 				elSelectWrapper.addEventListener("blur", self._blurBound);
 				if (!self.options.nativeMenu) {
 					elSelectWrapper.addEventListener("vclick", self._toggleMenuBound);
+					elOptionContainer.addEventListener("touchmove", self._touchMoveBound);
```

Comment:
Need to remove the added event listener when destroying the element.

Reply:
done